README.md

```diff
@@ -25,13 +25,19 @@
   npm install
   ```
 - Create your own GitHub app: [instructions](https://probot.github.io/docs/development/#configure-a-github-app)
-- Store the private key as `private-key.pem` in the repository’s directory
-- Start the app with `APP_ID=1234 npm start` where `1234` is your GitHub App’s ID
-- update your GitHub App’s `Webhook URL` to your localtunnel.me URL
+- On your local machine, copy `.env.example` to `.env`.
+- Go to [smee.io](https://smee.io) and click **Start a new channel**. Set `WEBHOOK_PROXY_URL` in `.env` to the URL that you are redirected to.
+- [Create a new GitHub App](https://github.com/settings/apps/new) with:
+  - **Webhook URL**: Use your `WEBHOOK_PROXY_URL` from the previous step.
+  - **Webhook Secret**: `development`.
+  - **Permissions & events** is located lower down the page and will depend on what data you want your app to have access to. Note: if, for example, you only enable issue events, you will not be able to listen on pull request webhooks with your app. However, for development we recommend enabling everything.
```

Comment:
This is for the local setup of WIP bot so we know what permission the app needs:
- Commit statuses (read & write)
- Pull Requests (read only)
- Subscribe to events: Pull request

Reply:
Ok. Will update that

Reply:
Ok. Will update that